### PR TITLE
Fix some remarks

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Tested. After opening motion create form I get still the error "Cannot read property 'value' of undefined". Can you please check it?

File: client/src/app/site/motions/motions.constants.ts [hunk 1]
```diff
@@ -68,31 +68,6 @@ export const verboseChangeRecoMode = {
     modified_final_version: 'Final print template'
 };
 
-/**
- * Enum to define different types of notifications.
- */
-export enum MotionEditNotificationType {
```
Comment (FinnStutzenstein) [inline client/src/app/site/motions/motions.constants.ts:74]: You deleted this enum - but where is the new one?

Comment (GabrielInTheWorld) [reply client/src/app/site/motions/motions.constants.ts:74]: I have changed it to `EditNotificationType`

Comment (FinnStutzenstein) [reply client/src/app/site/motions/motions.constants.ts:74]: Ah I oversaw these changes..

**Review by emanuelschuetze — APPROVED** [review]
Ok

**Review by FinnStutzenstein — APPROVED** [review]

**Pull request merged**